grails-forge-core/src/main/java/org/grails/forge/feature/test/Geb.java

```diff
@@ -105,24 +105,29 @@ public void apply(GeneratorContext generatorContext) {
                 .artifactId("geb")
                 .test());
```

Comment:
This should be included as `integrationTestImplementation testFixtures('org.grails.plugins:geb')`

However, to be able  to run the `create-functional-test` command (which has not been upgraded to a Grails 6 `GrailsApplicationCommand`), I'm not yet sure what needs to be put where. Is there any point in rewriting the `create-functional-test` command as a `GrailsApplicationCommand` now that the `grails-shell` cli is back?

Reply:
I'll test this with a generated app and then make the changes in forge.  It was not loading the transitive geb-spock dependency as is, but it wasn't using `integrationTestImplementation testFixtures`

Reply:
"integrationTestImplementation testFixtures('org.grails.plugins:geb' worked great and that scope is now an option in forge.

I think we should hold off command conversions.